content/livesync/connector/index.textile

```diff
@@ -0,0 +1,336 @@
+---
+title: Deploy
```

Comment:
I don't think this is the right terminology for this page. The "Deploy to Railway" heading is fine, as that is an actual example deployment to Railway cloud. However the rest of it is about running locally with docker compose (or just pulling the container image), not deployment.

Reply:
Run the Connector

Reply:
or install @mschristensen

Reply:
Fixed in this [commit](https://github.com/ably/docs/pull/2082/commits/15a2be26866049471006c24c9e1e9094c5b53a85)